web-common/src/app.css

```diff
@@ -226,5 +202,5 @@ body {
 }
 
 a {
-  @apply text-blue-500 hover:text-blue-600 font-medium;
+  @apply text-primary-500 hover:text-primary-600 font-medium;
```

Comment:
@jkhwu do we want <a> links to be `blue` (following standard web convention?) or `primary` (to match our theme)?

Reply:
@jkhwu bump on this...

Reply:
See this issue or this Notion doc for how I intend to color links: https://github.com/rilldata/rill/issues/3909

https://www.notion.so/rilldata/Incorporate-new-logo-and-colors-into-app-6bf813aff7c84b549fbd152d5c5f954b